kaspresso/src/main/kotlin/com/kaspersky/kaspresso/files/resources/impl/DefaultResourcesRootDirsProvider.kt

```diff
@@ -6,6 +6,8 @@ import java.io.File
 class DefaultResourcesRootDirsProvider : ResourcesRootDirsProvider {
     override val logcatRootDir = File("logcat")
     override val screenshotsRootDir = File("screenshots")
+    override val originalScreenshotsRootDir = File("original_screenshots")
+    override val screenshotsDiffRootDir = File("diff")
```

Comment:
It might be better to rename this diff file. The context of the screenshots is lost